docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -0,0 +1,233 @@
+
+* *geo.src*
+
+* *ip*
+
+.. In the editor, remove *Count of records*, then click *Save and return*.
+
+[float]
+==== Create the drilldown
+
+Create a drilldown that opens the *Host Overview* dashboard from the *[Logs] Web traffic* dashboard.
+
+. Open the panel menu for the data table you created, then select *Create drilldown*.
+
+. Click *Go to dashboard*.
+
+.. Give the drilldown a name. For example, `My Drilldown`.
+
+.. From the *Choose a destination dashboard* dropdown, select *Host Overview*.
+
+.. To use the geo.src filter, KQL query, and time filter, select *Use filters and query from origin dashboard* and *Use date range from origin dashboard*.
+
+.. Click *Create drilldown*.
+
+. Save the dashboard.
+
+. In the data table panel, hover over a value, click *+*, then select `My Drilldown`.
++
+[role="screenshot"]
+image::images/drilldown_on_panel.png[Drilldown on data table that navigates to another dashboard]
+
+[float]
+[[url-drilldowns]]
+=== Create URL drilldowns
+
+URL drilldowns enable you to navigate from a dashboard to external websites. Destination URLs can be dynamic, depending on the dashboard context or user interaction with a panel. To create URL drilldowns, you add <<variables,variables>> to a URL template, which configures the behavior of the drilldown.
+
+[role="screenshot"]
+image:images/url_drilldown_go_to_github.gif[Drilldown on pie chart that navigates to Github]
+
+Some panels support multiple interactions, also known as triggers. 
+The <<url-template-variables,variables>> you use to create a <<url_templating-language, URL template>> depends on the trigger you choose. URL drilldowns support these types of triggers:
+
+* *Single click* &mdash; A single data point in the panel.
+
+* *Range selection* &mdash; A range of values in a panel.
+
+For example, *Single click* has `{{event.value}}` and *Range selection* has `{{event.from}}` and `{{event.to}}`.
+
+The panels you create using the following editors support dashboard drilldowns:
+
+* *Lens*
+* *Maps*
+* *TSVB*
+* *Aggregation-based* area chart, data table, heat map, horitizontal bar chart, line chart, pie chart, tag cloud, and vertical bar chart
+
+[float]
+==== Create a URL drilldown
+
+For example, if you have a dashboard that shows data from a Github repository, you can create a URL drilldown that opens Github from the dashboard panel.
+
+. Add the *Sample web logs* data.
+
+. Open the *[Logs] Web traffic* dashboard.
+
+. In the toolbar, click *Edit*.
+
+. Open the *[Logs] Visitors by OS* panel menu, then select *Create drilldown*.
+
+. Click *Go to URL*.
+
+.. Give the drilldown a name. For example, `Show on Github`.
+
+.. For the *Trigger*, select *Single click*.
+
+.. To navigate to the {kib} repository Github issues, enter the following in the *Enter URL* field:
++
+[source, bash]
+----
+https://github.com/elastic/kibana/issues?q=is:issue+is:open+{{event.value}}
+----
++
+`{{event.value}}` is substituted with a value associated with a selected pie slice.
```

Comment:
Not sure if this is something the user needs to do, or if this is what Kibana does.